README.adoc

```diff
@@ -1,3 +1,22 @@
+= C++ example project scanned on SonarQube using GitHub Actions
```

Comment:
Please, mention in the readme that the `build.yml` is not complete and the user needs to set the correct sonarqube URL (maybe give an example of a correct URL)